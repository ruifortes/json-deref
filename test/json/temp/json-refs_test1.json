{
  "a": {"$ref": "#/b"},
  "b": {"$ref": "#/c"},
  "c": "value of c"
}
