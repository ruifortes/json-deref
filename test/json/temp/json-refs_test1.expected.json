{
  "a": "value of c",
  "b": "value of c",
  "c": "value of c"
}
